docker-compose.yml

```diff
@@ -138,5 +138,10 @@ services:
   redis:
     image: redis:3
 
+  mailcatcher:
```

Comment:
Will this brought up with `make run`? If not let's add it there too

Reply:
Do we want it to come up by default? Since using it requires you to customise your environment variables, I'd leave it as optional.

Reply:
Let's also add these vars with some default values in the env file